app/src/androidTest/java/com/nervousfish/nervousfish/test/LoginActivitySteps.java

```diff
@@ -35,39 +34,24 @@ public LoginActivitySteps(LoginActivity activityClass) {
     }
 
     @Given("^I have a LoginActivity")
-    public void I_have_a_LoginActivity() {
+    public void i_have_a_LoginActivity() {
         assertNotNull(getActivity());
     }
 
-    @When("^I input email (\\S+)$")
-    public void I_input_email(final String email) {
-        onView(withId(R.id.email)).perform(typeText(email));
-    }
-
     @When("^I input password \"(.*?)\"$")
-    public void I_input_password(final String password) {
+    public void i_input_password(final String password) {
         onView(withId(R.id.password)).perform(typeText(password));
     }
 
     @When("^I press submit button$")
-    public void I_press_submit_button() {
+    public void iPressSubmit() {
         onView(withId(R.id.submit)).perform(scrollTo()).perform(click());
     }
 
-    @Then("^I should see error on the (\\S+)$")
-    public void I_should_see_error_on_the_editTextView(final String viewName) {
-        int viewId = (viewName.equals("email")) ? R.id.email : R.id.password;
-        int messageId = (viewName.equals("email")) ? R.string.msg_email_error : R.string.msg_password_error;
-
-        onView(withId(viewId)).check(matches(hasErrorText(getActivity().getString(messageId))));
-    }
-
     @Then("^I should (true|false) auth error$")
     public void I_should_see_auth_error(boolean shouldSeeError) {
         if (shouldSeeError) {
             onView(withId(R.id.error)).check(matches(isDisplayed()));
-        } else {
```

Comment:
I still think that is true, but that doesn't apply here.

This test is named "I_should_see_auth_error" so that is wat it should test, this is not at all related to switching to a different activity. So:
- If you're using this test to see if the activity changes, you're using it incorrectly and you should change that. Or,
- if you're not using this test for that, you should still keep the `else` branch because, as I stated before, the description/annotation of the test still describes a situation where the else branch is needed.